### DISCUSSION TITLE
How to get custom text entered by user on the particular scene?

### DISCUSSION THREAD

**Discussion by lofti198** [telegraf/telegraf#1607] [Q&A] [answered]
Hi! I found scene.hears function, however as far as I understand its intended to catch predefined commands from user. How can I get his input (for instance his surname) during the particular scene?

**Comment by lofti198** [comment, marked as answer]
found solution customScene.on('text', ctx =>{ ctx.reply(ctx.message.text);})

**Reply by humblecoder** [reply]
This only works if the bot is _**Admin**_, no?  

Are there any quality options for **_non-admin_** bots (that are using InlineKeyboards) to acquire a "direct input/response" as the result of a button press?

**Reply by MKRhere** [reply]
@humblecoder The bot doesn't need to be admin to receive messages, it needs to have "Group privacy" disabled with botfather.
